packages/core/src/index.ts

```diff
@@ -59,23 +58,40 @@ import {
   pickVersion
 } from '@hoprnet/hopr-utils'
 
+import {
+  core_hopr_initialize_crate,
+  core_hopr_gather_metrics,
+  CoreApp,
+  HeartbeatConfig,
+  Database,
+  Address as Packet_Address,
+  PacketInteractionConfig,
+  Path,
+  Payload,
+  PublicKey as Packet_PublicKey,
+  Balance as Packet_Balance,
+  WasmNetwork,
+  WasmPing,
+  WasmAckInteraction,
+  WasmPacketInteraction
+} from '../lib/core_hopr.js'
+core_hopr_initialize_crate()
+registerMetricsCollector(core_hopr_gather_metrics)
+
 import { FULL_VERSION, INTERMEDIATE_HOPS, MAX_HOPS, PACKET_SIZE, VERSION, MAX_PARALLEL_PINGS } from './constants.js'
 
 import {
-  Network,
   PeerStatus,
   PeerOrigin,
   Health,
   health_to_string,
-  HeartbeatConfig,
   core_network_initialize_crate,
-  core_network_gather_metrics
+  core_network_gather_metrics,
+  PingConfig
 } from '../lib/core_network.js'
```

Comment:
Might be worth to re-export `core-network` through `core-hopr` like we do with e.g. `core-path`:
````
    // Temporarily re-export core-path
    #[allow(unused_imports)]
    use core_path::wasm::*;
 ````

So conversions could be avoided.